core/src/test/scala/com/mesosphere/usi/core/scalaapi/SchedulerAdapterTest.scala

```diff
@@ -0,0 +1,315 @@
+package com.mesosphere.usi.core.scalaapi
+
+import akka.NotUsed
+import akka.stream.scaladsl.{SinkQueueWithCancel, SourceQueueWithComplete}
+import akka.stream.scaladsl.{Flow, Sink, Source}
+import akka.stream.OverflowStrategy
+import com.mesosphere.usi.core.Scheduler
+import com.mesosphere.usi.core.models.resources.ScalarRequirement
+import com.mesosphere.usi.core.models._
+import com.mesosphere.utils.AkkaUnitTest
+import org.scalatest._
+
+import scala.concurrent.Promise
+import scala.util.{Failure, Success}
+
+class SchedulerAdapterTest extends AkkaUnitTest with Inside {
+
+  def createMockedScheduler: (
```

Comment:
Don't we have such a mock in `test-utils`, @alenkacz and @zen-dog?

Reply:
I think we have one for mesos, not for scheduler as a whole... but I might be mistaken :)